tuareg.el

```diff
@@ -1136,10 +1136,13 @@ Regexp match data 0 points to the chars."
 (defvar tuareg-mode-abbrev-table ()
   "Abbrev table used for Tuareg mode buffers.")
 
+(defvar tuareg-abbrev-hook nil
+  "Abbrev hook for tuareg mode buffers.")
+
 (if tuareg-mode-abbrev-table ()
   (define-abbrev-table 'tuareg-mode-abbrev-table
     (mapcar (lambda (keyword)
-              `(,keyword ,keyword tuareg-abbrev-hook nil t))
+              `(,keyword ,keyword (lambda () (run-hooks tuareg-abbrev-hook)) nil t))
```

Comment:
Do we want to use `run-hook-with-args` and pass `keyword` to the hooks, in which case the `run-hooks` should be run at definition time — and not every time an abbrev is triggered?